ports/nrf/boards/arduino_nano_33_ble/mpconfigboard.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2022 Arduino SA
+ */
+
+#define MICROPY_HW_BOARD_NAME       "Arduino Nano 33 BLE SENSE"
+#define MICROPY_HW_MCU_NAME         "NRF52840"
+#define MICROPY_PY_SYS_PLATFORM     "nano33"
+
+#define MICROPY_MBFS                (1)
+
+#define MICROPY_BOARD_EARLY_INIT NANO33_board_early_init
+void NANO33_board_early_init(void);
+
+#define MICROPY_BOARD_DEINIT NANO33_board_deinit
+void NANO33_board_deinit(void);
+
+#define MICROPY_BOARD_ENTER_BOOTLOADER(nargs, args) NANO33_board_enter_bootloader()
+void NANO33_board_enter_bootloader(void);
```

Comment:
To make this a bit neater, I suggest moving all the function declarations down to the bottom of this file.  Ie here you want to have:
```
#define MICROPY_BOARD_EARLY_INIT NANO33_board_early_init
#define MICROPY_BOARD_DEINIT NANO33_board_deinit
#define MICROPY_BOARD_ENTER_BOOTLOADER(nargs, args) NANO33_board_enter_bootloader()
```
and then at the bottom of the file:
```
void NANO33_board_early_init(void);
void NANO33_board_deinit(void);
void NANO33_board_enter_bootloader(void);
```